### DISCUSSION TITLE
Theme Store Idea

### DISCUSSION THREAD

**Discussion by du82** [status-im/status-legacy#16650] [General]
As a user / creator, I'd love to customize the app beyond 12 colors. I'd also like a store where users can share theme packs that lets you preview how it will look in-app before applying it.

### Rationale:
Users feel more attached to things that they can customize to their liking. Notion has capitalized off of this, as well as every popular Chinese app has some sort of customization beyond light/dark mode. By offering a flexible themeing system that lets users make the app look the way they want, they will likely enjoy it more since most western apps haven't caught on just yet.

Baidu has this functionality already, and it's a react native  (JavaScript) app. Examples below:

<img src="https://user-images.githubusercontent.com/65834069/225823600-01cde52d-f8dd-4597-992c-4caf32402ee8.jpg" width=30%>
<img src="https://user-images.githubusercontent.com/65834069/225823602-3568c3db-dcc7-44cd-9279-38ef87b505e5.jpg" width=30%>
<img src="https://user-images.githubusercontent.com/65834069/225823607-0b5f26f5-19af-4b2f-8043-5e3713797cf3.jpg" width=30%>

### Profile cuatomization
To improve user stickiness, we should provide an easy way to customize your profile beyond just a profile pic and background color.

Baidu has a store for this where designers can share their profile customizations.

<img src="https://user-images.githubusercontent.com/65834069/225823809-a0f4d72f-0b3d-4512-b3b1-b4a3a2bb6207.jpg" width=30%>
<img src="https://user-images.githubusercontent.com/65834069/225823813-04d0ae79-fc8b-4c18-a352-1ac79a0f9de7.jpg" width=30%>

**Comment by flexsurfer** [comment]
cc @John-44
